package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "postinstall": "npx patch-package && cd desktop && npm install",
+    "postinstall": "cd desktop && npm install",
```

Comment:
@dhairyasenjaliya, recently https://github.com/Expensify/App/pull/12864 was merged where we added another patch, so we shouldn't delete npx patch-package from postinstall